cmd/utils/utils.go

```diff
@@ -325,10 +307,14 @@ func renderUpdatedTable(w io.Writer, updatedAt time.Time) error {
 	return tw.Flush()
 }
 
-func zeroOfPtr[T comparable](v *T) T {
+func ZeroOfPtr[T comparable](v *T) T {
 	var zero T
 	if v == nil {
 		return zero
 	}
 	return *v
 }
+
+func PtrBytes(v []byte) *[]byte {
```

Comment:
This should not be required now that we have generics.

Reply:
we said we leave this for later